packages/unmock-core/src/service/spy/index.ts

```diff
@@ -0,0 +1,63 @@
+import { SinonSpy, spy as sinonSpy } from "sinon";
+import { ISerializedRequest, ISerializedResponse } from "../../interfaces";
+
+export type RequestResponseSpy = SinonSpy<
+  [ISerializedRequest],
+  ISerializedResponse
+>;
+
+interface IRequestResponsePair {
+  req: ISerializedRequest;
+  res: ISerializedResponse;
+}
+
+export interface ICallTracker {
+  /**
+   * Keep track of calls, added via the `notify` method
+   */
+  spy: RequestResponseSpy;
+  /**
+   * Add new call to the spy object
+   */
+  track(pair: IRequestResponsePair): void;
+  reset(): void;
+}
+
+export type RequestResponseSpyNotifier = CallTracker<
```

Comment:
Why are the above interfaces/types exported? :thinking:

Reply:
Also this specific type (`RequestResponseSpyNotifier`) is not used anywhere at the moment.

Reply:
I just think it's cleaner to export an interface instead of concrete class.

Reply:
And yes that `RequestResponseSpyNotifier` was a left-over, removed.